spec/unit/shoulda/matchers/active_record/have_rich_text_matcher_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'unit_spec_helper'
+
+describe Shoulda::Matchers::ActiveRecord::HaveRichText, type: :model do
+  shared_examples 'undefined rich text' do |kind|
```

Comment:
I like that you're trying to keep things DRY in this file, but since each of these shared example groups only have one test in them, I feel like it actually makes things a bit harder to follow when reading over these tests. Would you mind unfolding them? It doesn't feel like a lot of duplication anyway.

Reply:
Done!